l10n_cl_sii_folio/models/account_move.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (C) 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, models, _
+from odoo.exceptions import UserError
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    @api.multi
+    def post(self, invoice=False):
```

Comment:
Max, this is overriding the post() method, breaking the inheritance chain.
I just found a bug on another module because if this.
Can we call super() somewhere?

Reply:
@dreispt We call super() line 16 for all the other journals than sale and purchase. If we call super() for those 2, we cannot use the document to select which sequence to use and get the number.